.jenkins

```diff
@@ -0,0 +1,35 @@
+pipeline {
+    agent {
+        label 'DoNotRunAnythingOnMaster'
+    }
+    stages {
+        stage("Deploy documentation") {
+            when {
+                 branch "master"
+            }
+            agent {
+                docker {
+                    image "ubuntu:18.04"
+                    label 'docker'
+                }
+            }
+            environment {
+                GIT_CREDS = credentials('0f170146-bd5e-44c0-a6e0-4f6fac3230fa')
+            }
+            steps {
+                sh 'apt update && apt install --no-install-recommends -y build-essential cmake doxygen git'
```

Comment:
We also need `graphviz` to get the Doxygen dot module for the formulas.

Also, are we rebuilding the docker image on every pull-request? Seems slow and wasteful.

Reply:
I was too lazy to define a Dockerfile but I will add it

Reply:
Done.